src/DataStructures.jl

```diff
@@ -102,6 +104,8 @@ module DataStructures
 
     include("priorityqueue.jl")
 
+    include("sorted_vector.jl")
```

Comment:
Funny, my comment was based on `priorityqueue.jl`, which is the only example shown in the diff, and apparently also the only exception.